core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -315,11 +321,14 @@ object ConsoleConsumer extends Logging {
 
     formatter.configure(formatterArgs.asScala.asJava)
 
-    val topicOrFilterOpt = List(topicOpt, whitelistOpt).filter(options.has)
+    val topicOrFilterOpt = List(topicOpt, whitelistOpt, includeOpt).filter(options.has)
     if (topicOrFilterOpt.size != 1)
-      CommandLineUtils.printUsageAndDie(parser, "Exactly one of whitelist/topic is required.")
+      CommandLineUtils.printUsageAndDie(parser, "Exactly one of include/topic is required.")
```

Comment:
I would still mention `whitelist` here as it is still usable.

Reply:
I'm not sure if it makes sense to list deprecated configurations in error messages, do we do this elsewhere? The option would still show up in the CLI help.

Reply:
I don't know if we did this somewhere else.

I just think that the error message is confusing for folks who would use the deprecated argument. Say you provide the deprecated one with topic and you get an error which mention the new one. This is confusing.

As an alternative, we could actually mention the one that was used in the error message.

Reply:
We can have `include/whitelist/topic` and remove it when we completely remove `--whitelist` later.

Reply:
I agree with @xvrl it doesn't make sense to list deprecated config. Especially that this error message will be displayed only if `--whitelist`, `--include` and `--topic` are all empty.

Reply:
The error message now will be `Exactly one of include/topic is required. whitelist is DEPRECATED use --include instead.` if there's any filter specified (`whitelist` or `include` with `topic`) and If a user set value for both `whitelist` and `include` there will be a warning message `--whitelist is DEPRECATED and ignored if --include specified.`